README.md

```diff
@@ -4,32 +4,41 @@
 
 # cbuild: Open-CMSIS-Pack Build Invocation Utility
 
-This utility allows embedded developers to orchestrate the build of CPRJ projects using `cbuildgen`, `cpackget`, `cmake` and `ninja.`
+This utility allows embedded developers to orchestrate the build of CPRJ and CSolution projects using `cbuildgen`, `csolution`, `cpackget`, `cmake` and `ninja.`
```

Comment:
This utility allows embedded developers to build CPRJ and csolution projects by orchestrating the following tools: ...